config.xml

```diff
@@ -80,6 +80,10 @@
     <plugin name="cordova-plugin-device" spec="^2.0.1" />
     <plugin name="cordova-plugin-splashscreen" spec="^5.0.2" />
     <plugin name="cordova-plugin-ionic-webview" spec="^1.2.0" />
-    <plugin name="cordova-plugin-ionic-keyboard" spec="^2.0.5" />
     <plugin name="cordova-plugin-inappbrowser" spec="~3.0.0" />
+    <allow-navigation href="http://10.201.82.242:8100" />
+    <plugin name="cordova-plugin-iroot" spec="^0.7.0" />
```

Comment:
Quick question. Are we planning to wrap those plugins at some point to reflect what we done with Core (and Auth in future)?

Reply:
@cianclarke Cordova JS provides now means for this usage and we spiked this already. Let's sync on this on the other PR.